test/unit/socket-draft-dgram-test.ts

```diff
@@ -5,60 +5,61 @@ import {assert} from "chai"
 import {createSocket} from "dgram"
 import {testProtos, uniqAddress} from "./helpers"
 
-if (zmq.capability.draft) {
-  for (const proto of testProtos("udp")) {
-    describe(`draft socket with ${proto} dgram`, function () {
-      let dgram: draft.Datagram
+for (const proto of testProtos("udp")) {
+  describe(`draft socket with ${proto} dgram`, function () {
+    if (zmq.capability.draft !== true) {
+      if (process.env.ZMQ_DRAFT === "true") {
+        throw new Error("Draft API requested but not available at runtime.")
+      }
+      return
+    }
 
-      beforeEach(function () {
-        dgram = new draft.Datagram()
-      })
+    let dgram: draft.Datagram
 
-      afterEach(function () {
-        dgram.close()
-        global.gc?.()
```

Comment:
Good! It makes me uncomfortable that we're calling `global.gc` all over the place without any seeming justification.